helios/pipeViewer/transactionsearch/src/transaction_search.cpp

```diff
@@ -0,0 +1,280 @@
+/*
+ */
+
+#include "transactiondb/src/Reader.hpp"
+#include "transactiondb/src/PipelineDataCallback.hpp"
+#include <iostream>
+#include <string>
+#include <set>
+#include <boost/regex.hpp>
+
+
+#define RESULT_TAG 'r'
+#define PROGRESS_TAG 'p'
+#define INFO_TAG 'i'
+
+#define START_DELIMITER ':'
+
+#define NUMBER_OF_PROGRESS_UPDATES 50
+
+/*! Stores regular expression for comparison in callbacks */
+boost::regex global_regular_expression;
```

Comment:
unless you're using perl-style regexes, consider std::regex, using std::regex can avoid having a dependency on boost::regex

Reply:
Agreed.  Removed.